gloo/transport/uv/libuv.h

```diff
@@ -280,13 +284,21 @@ class Resource : public Emitter<T>, public std::enable_shared_from_this<T> {
 
   template <typename R>
   const R* get() const noexcept {
+  #ifdef _WIN32
+    static_assert(!std::is_same<R, U>::value, "!");
+  #else
     static_assert(not std::is_same<R, U>::value, "!");
+  #endif
```

Comment:
Use the `!` version for all platforms, here and in all locations below, no need to break it out like this. They are lexically identical, the text form is just a legacy of C that really needs to go away.